INSTALL.md

```diff
@@ -35,12 +35,17 @@ Note: the Intel(R) oneAPI components listed here can be installed together throu
 
 https://www.intel.com/content/www/us/en/developer/tools/oneapi/base-toolkit.html
 
+These dependencies can also be installed through the `conda` software, but doing so will require a few additional setup steps - see rest of the instructions for details.
+
 ## Docker Development Environment
 
 [Docker file](https://github.com/uxlfoundation/oneDAL/tree/main/dev/docker) with the oneDAL development environment
 is available as an alternative to the manual setup.
 
 ## Installation Steps
+
+_(If using `conda`, see subsequent sections for instructions)_
```

Comment:
I don't think this is needed if the reference to the chapter is added at the beginning.

Reply:
Removed.